pkgs/by-name/va/vala-panel-appmenu/package.nix

```diff
@@ -0,0 +1,131 @@
+# to prevent bloating this package, please choose what integration you want
+# to use manually, like this:
+#   vala-panel-appmenu.override { enableMate = true; }
+# the package compiles very quickly, so there is no sense to have every
+# single combination to be build by hydra
+{
+  lib,
+  budgie-desktop,
+  fetchFromGitLab,
+  glib,
+  gobject-introspection,
+  gtk2,
+  jdk,
+  libdbusmenu,
+  libwnck,
+  libxkbcommon,
+  mate,
+  meson,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  stdenv,
+  stripJavaArchivesHook,
+  vala,
+  vala-panel-appmenu,
+  wrapGAppsHook3,
+  xfce,
+
+  enableXfce ? false,
+  enableBudgie ? false,
+  enableValapanel ? false, # NOTE: requires https://github.com/rilian-la-te/vala-panel which is not packaged
```

Comment:
If it isn't packaged, how will setting this flag to true ever work? Edit: I see you put it into `meta.broken`, but still that means that the option is practically unused...

Reply:
One may be very confused why the feature is in the upstream, but not available in nixpkgs. This makes it clear, if someone would want valapanel feature from this package, but couldn't find anything about it in this file

Reply:
This would require one to go to the source code (this file) and read it, instead right now it raises an error with then easily explainable on the first lines of the file.

Reply:
My reasoning here would be: how would one know about the override option if they haven't read the file? But I think our opinions differ so little that I'd write this one off as bikeshedding (sorry for the inconvenience). I see you addressed my other comments, so I'll just approve you here.